src/mainviews/detail-view/job-definition.tsx

```diff
@@ -61,13 +62,11 @@ export function JobDefinition(props: IJobDefinitionProps): JSX.Element {
           {trans.__('Resume')}
         </Button>
       )}
-      <Button
-        variant="contained"
-        color="error"
-        onClick={handleDeleteJobDefinition}
-      >
-        {trans.__('Delete Job Definition')}
-      </Button>
+      <DeleteWithConfirmationButton
+        handleDelete={handleDeleteJobDefinition}
+        title={trans.__('Delete Job Definition')}
+        text={trans.__('Are you sure you want to delete this Job Definition?')}
```

Comment:
"Job" and "job definition" are not proper nouns, and so should only be capitalized at the start of a sentence.

```suggestion
        text={trans.__('Are you sure that you want to delete this job definition?')}
```

Reply:
I've been capitalizing job and job definition every time for the past 1.5 months 💀